backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/pporder/api/impl/AbstractPPOrderReceiptHUProducer.java

```diff
@@ -98,6 +104,8 @@
 
 /* package */abstract class AbstractPPOrderReceiptHUProducer implements IPPOrderReceiptHUProducer
 {
+	private final static AdMessageKey MESSAGE_Hergestellt = AdMessageKey.of("Hergestellt");
```

Comment:
why not calling the message key `ClearanceStatusInfo.Manufactured` (or something like that)?

Reply:
makes sense, changed.